doc/en/how-to/output.rst

```diff
@@ -270,6 +270,10 @@ situations, for example you are shown even fixtures that start with ``_`` if you
 Using higher verbosity levels (``-vvv``, ``-vvvv``, ...) is supported, but has no effect in pytest itself at the moment,
 however some plugins might make use of higher verbosity.
 
+By default for verbosity inferior to ``-vv`` really long output are truncated. It's also possible to
```

Comment:
This is great, but in addition to this, we should also include this option in the `Configuration Options` section in `reference.rst`, making use of the `confval` directive, for example:

https://github.com/pytest-dev/pytest/blob/f6adebb99064ad2a894ea7c04a1257aba5876e21/doc/en/reference/reference.rst?plain=1#L1180-L1184

(Make sure to include the new option in alphabetical order)